design/README.md

```diff
@@ -1,34 +1,30 @@
 
-## Design Approval
+If you are comfortable using GitHub and the command line on your computer, you can follow the instructions below for creating your own Pull Request. 
 
-Designers can freely create icons for their project in their own feature branch, but in order to incorporate into the design system a pull request must be created and approved. See Git Steps for Designers below for more information.
+> If you're not familiar with GitHub and/or the command line, you can also send us your icons directly via email and we will create the Pull Request for you and relay any feedback.
 
-Two reviews will be required to approve the pull request and merge into the system. A design review by a Senior designer to ensure the icon follows Material and PXBlue standards and a code review to ensure the SVG is formated properly. Reviewers must also agree that the name of the icon makes sense, avoids collisions, and that the icon fits into the proposed family folder.
+Before acceptance, new icons must be reviewed by:
+- A Senior PX Blue designer to ensure that the icons are properly following Material Design guidelines
+- A PX Blue developer to ensure that the SVG files are properly formatted and function correctly with the build pipeline
 
-## Design Creation
+Reviewers must also agree that the name of the icon makes sense, avoids collisions, and that the icon fits into the proposed family folder.
 
-[Follow Material Design icon design guidelines & principles](https://material.io/guidelines/style/icons.html#icons-system-icons). Icons will be compared against these guideslines for approval.
+## Icon Creation
 
-Templates are also provided to assist in the creation of icons.
+When creating icons, follow the Material Design [icon design guidelines](https://material.io/guidelines/style/icons.html#icons-system-icons). Icon designs that do not adhere to these guidelines will be rejected. Templates are available to assist you:
+-   Illustrator: _IconTemplate.ait_
+-   Printable: _MDILetter.pdf_, _MDITabloid.pdf_
 
-#### Template Files
+Design files should ideally contain both original paths, strokes, text, etc. (in the case that the icon must be modified to increase stroke width or font size, for example) as well as the expanded paths version. The expanded version should be use to generate the SVG file.
 
--   Illustrator - _IconTemplate.ait_
--   Sketch - _to do_
--   Printable - _MDILetter.pdf_, _MDITabloid.pdf_
-
-As noted above, use the provided AI and printable template files to design your icons. The source file should ideally contain both original paths, strokes, text, etc. (in the case that the icon must be modified to increase stroke width or font size, for example) as well as the expanded paths version.
-
-The expanded version should be use to generate the SVG file.
-
-#### SVG Export Guidelines
+### SVG Export Guidelines
 
```

Comment:
In Adobe Illustrator, go to File > Export > Export As and use the following export options: ...

There are three export options in AI